cloud/terraform/aws_config_builder.py

```diff
@@ -69,7 +70,7 @@ def __new_aws_instance(self, instance):
             'ami': instance['ami'],
             'provider': f'aws.{instance["region"]}',
             'key_name': instance['aws_key_pair'],
-            'tags': {'name': name_tag},
+            'tags': {'name': name_tag, 'gitlab-ci-test': 'true'},
```

Comment:
This tag will be shared among all cloud providers. Right?

If that will be the case, I suggest adding an attribute to the BaseConfigBuilder class and use it here as a key.

E.g.:
```suggestion
            'tags': {'name': name_tag, self.ci_tag_key: 'true'},
```